Neo4j.Driver/Neo4j.Driver/Internal/Connector/SocketConnection.cs

```diff
@@ -171,7 +171,13 @@ public void Reset()
 
         public bool IsOpen => _client.IsOpen;
         public IServerInfo Server { get; set; }
-        public IBoltProtocol BoltProtocol => _boltProtocol;
+
+        public IBoltProtocol BoltProtocol
```

Comment:
Do we need this change, as far as I see, we're only reading this from outside?

Reply:
It was used for test only. I want to pass in a mocked protocol. Without a setter, the only way to set it was via a success Conn.Init. As this class is internal so I directly added a setter.